Privacy Page UI

song50119 opened the pull request:
Resolves #292

Sirmorrison commented on cli/termio/terminalprompt/getinput.go:
```diff
@@ -38,51 +67,83 @@ func getTextInput(prompt string) (string, error) {
-		<-signalChannel
-		setTerminalEcho(true)
-	}()
+	psw, err := getPasswd(prompt, true, os.Stdin, os.Stdout)
```
use `getPassword` instead.

Sirmorrison commented on cli/termio/terminalprompt/getinput.go:
```diff
@@ -38,51 +67,83 @@ func getTextInput(prompt string) (string, error) {
+// If prompt is not empty, it will be output as a prompt to the user
+// If masked is true, typing will be matched by asterisks on the screen.
+// Otherwise, typing will echo nothing.
+func getPasswd(prompt string, masked bool, r FdReader, w io.Writer) ([]byte, error) {
```
`getPassword`

Sirmorrison commented on cli/termio/terminalprompt/getinput.go:
```diff
@@ -38,51 +67,83 @@ func getTextInput(prompt string) (string, error) {
+// Otherwise, typing will echo nothing.
+func getPasswd(prompt string, masked bool, r FdReader, w io.Writer) ([]byte, error) {
+	var err error
+	var pass, bs, mask []byte
```
what is `bs` please. `pass` do you mean `password` if yes please rename.

song50119 committed:
Adds Privacy Page layout

oshorefueled commented on ui/privacy_page.go:
```diff
@@ -0,0 +1,232 @@
+			)
+		})
+	})
+}
```
This layout is also used by the settings page. You can create a subpage layout with this header layout and wrap the privacy page in the subpage layout. There's already a subpage layout implemented in pages.go used by the verify message and sign message pages. You can create another one named "subpageSplitLayout"

song50119 committed:
Update Privacy Page
- add wallet menu privacy item
- create subpage layout for settings page
- add resuable headers for subpage layouts
- implement dangerZoneLayout collapsible

oshorefueled commented on ui/page.go:
```diff
@@ -687,6 +693,32 @@ type SubPage struct {
+	})
+}
+
+func (page pageCommon) subPageHeader(gtx layout.Context, sp *SubPage) layout.Dimensions {
```
Why's this method taking a pointer as an argument? I don't see anywhere in the code where a property is being set.

song50119 committed:
Update Privacy Page
* Create privacyInfo modal
* Ability to add custom modal title
